package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
@orbeckst that's what I said. The changes were too substantial to warrant a point release. So, there's nothing to worry about with why these all went from being deprecations of things that will go missing in 0.15.0 to things that will go missing in 0.16.0. We just shifted the big #363 merge a release.